sources/packages/web/src/components/common/applicationTracker/ApplicationInProgressChangeRequest.vue

```diff
@@ -0,0 +1,216 @@
+      await router.push({
+        name: StudentRoutesConst.DYNAMIC_FINANCIAL_CHANGE_REQUEST_APP_FORM,
+        params: {
+          selectedForm: applicationProgramYear.formName,
+          programYearId: applicationProgramYear.programYearId,
+          id: applicationId,
+        },
+      });
+    };
+
+    const cancelChangeRequest = async () => {
+      if (!props.changeRequest) {
+        return;
+      }
+      const confirm = await cancelChangeRequestModal.value.showModal();
+      if (!confirm) {
+        return;
+      }
+      try {
+        cancelChangeRequestLoading.value = true;
+        await ApplicationService.shared.applicationCancelChangeRequest(
+          props.changeRequest.applicationId,
+        );
+        emit("changeRequestCancelled");
+      } catch {
+        snackBar.error(
+          "There was an error cancelling your change request. Please try again.",
+        );
+      } finally {
+        cancelChangeRequestLoading.value = false;
+      }
+    };
+
+    /**
+     * Creates the list of processes currently waiting that are
+     * meaningful to be displayed to the student.
+     */
+    const waitingList = computed(() => {
+      if (!props.changeRequest) {
+        return [];
+      }
+      const waitingList: WaitingTypes[] = [];
+      const change = props.changeRequest;
+      if (
+        change.applicationEditStatus ===
+        ApplicationEditStatus.ChangePendingApproval
+      ) {
+        waitingList.push(WaitingTypes.MinistryApproval);
+      }
+      if (
+        [change.parent1Info, change.parent2Info].includes(
+          SuccessWaitingStatus.Waiting,
+        )
+      ) {
+        waitingList.push(WaitingTypes.ParentsDeclaration);
+      }
+      if (change.partnerInfo === SuccessWaitingStatus.Waiting) {
+        waitingList.push(WaitingTypes.PartnerDeclaration);
+      }
+      if (
+        change.studentIncomeVerificationStatus === SuccessWaitingStatus.Waiting
+      ) {
+        waitingList.push(WaitingTypes.StudentIncomeVerification);
+      }
+      if (
+        [
+          change.parent1IncomeVerificationStatus,
+          change.parent2IncomeVerificationStatus,
+        ].includes(SuccessWaitingStatus.Waiting)
+      ) {
+        waitingList.push(WaitingTypes.ParentsIncomeVerification);
+      }
+      if (
+        change.partnerIncomeVerificationStatus === SuccessWaitingStatus.Waiting
+      ) {
+        waitingList.push(WaitingTypes.PartnerIncomeVerification);
+      }
+      return waitingList;
+    });
+
+    /**
+     * Adjust the label based on the waiting list.
+     */
+    const trackerLabel = computed(() => {
+      return waitingList.value.length
+        ? "You have a submitted change request that is still pending. Please see below for the next steps."
+        : "You have a submitted change request that is still pending.";
+    });
+
+    return {
+      SuccessWaitingStatus,
+      ApplicationEditStatus,
```

Comment:
ApplicationEditStatus is not required in the template.

Reply:
`SuccessWaitingStatus` and `ApplicationEditStatus` were removed.